index.bs

```diff
@@ -3174,6 +3188,13 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     1. Run [=obtain and deliver debug reports on source registration=] with
         « "<code>[=source debug data type/source-channel-capacity-limit=]</code>" » and |source|.
     1. Return.
+1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
```

Comment:
Let's reduce the scope of this variable since it's only needed when the `if` branch on the following line is taken.